proposals/stage-0/individual-membership-director/README.md

```diff
@@ -0,0 +1,73 @@
+- Coordinate at least one annual event that interfaces the Board of Directors with the Individual Membership class(ex: Town Hall) 
+- Provide feedback on how proposed board items might impact the $FOUNDATION projects ecosystem represented as individual members and can provide suggestions on how the $FOUNDATION can better support the community. Coordinate with the Foundation to contribute the resources the community needs to maintain and grow a healthy relationship with the ecosystem. 
+- Solicit feedback and data that represents the wishes of other individual members and the community at large with the assistance of the CPC. They have been entrusted with the duty to make decisions based on the information they receive to best represent the community, and can gather input for proposals when relevant and granted permission to do so. 
+- Participate and vote in Board meetings. This includes legal considerations, budgeting and approving Foundation-led conferences and other initiatives.  
+- Introduce and drive forward initiatives to conclusion that capture the mission of the Node.js project while taking advantage of the resources the Foundation has at its disposal, whether community or executive. 
+- Represent the Board at speaking engagements  
+- Participate and observe relevant $FOUNDATION ecosystem meetings in order to deliver monthly updates to the Board.  
+- Deliver monthly documented updates to the members of the Individual Membership class on what the Board and Foundation are doing. They have been entrusted with the duty to make decisions based on the information they receive to best represent the community, and can gather input for proposals when relevant and granted permission to do so by the Board. 
+
+### Required Resources
+
+What resources would be needed to execute it?
+
+- Administrator for elections, tooling, and communications coordination for membership with Director 
+- Tooling: membership management system and election system 
+
+### Who would be responsible?
+
+The CPC would own the maintenance and sustainability of the programs along with the Individual Membership Director and would be aided by the Executive group with resources of the $FOUNDATION to execute.
+
+### How would success be measured?
+
+Given the goals outlined in this proposal,
+
+- NPS score improves by 10% after base is established 
+    - With the introduction of better feedback mechanisms, we should be able to move some detractors to passives, and some passives to promoters. 
+
+- Annual survey from membership reflects confidence in Director as effective 
+
+### Why this proposal is important
+
+With the historical data we have regarding individual membership, if Board Directors wish for the Individual Membership to be representative of the larger Node.js engaged ecosystem, we know that the number and makeup of the prior votes was not statistically significant nor representative. The Individual Membership did not have a clear path, role, or definition. 
```

Comment:
There are a couple call outs to Node.js specifically. Should those be replaced for a term that represents all the projects in the foundation?

Reply:
what other JS platform would we include? All projects in the foundation are JS projects; are there any for which node isn’t relevant?

Reply:
Browsers comes to mind, but we don't need to call out any specifically 😊
I think "larger JavaScript engaged ecosystem" is more representative, there are many projects in the foundation that do not require Node.JS to run (moment, lodash, amp and I'm sure many more).

Reply:
all of them require node.js to build, and all distribute a node module, so node is relevant to them as well.

Is there any project that is not impacted by node? Is there any other platform that impacts every project?

I suspect the answer to both is "no".

Reply:
> so node is relevant to them as well.

Yes, but JS is the common denominator and here we are calling out particularly Node.JS and not JS in general. I believe there's a significant difference between needing Node.JS to build (which only applies to the developers) and Node.JS to run or use it (which applies to consumers which is several orders of magnitude bigger).

> to be representative of the larger Node.js engaged ecosystem

This phrase makes me think about pushing just for Node.JS and not JavaScript overall and we are in the OpenJS Foundation.

Reply:
JavaScript is node and browsers, primarily. There’s other engines, and they matter, but it would be shortsighted of the foundation to ignore the dominant server/build platform in the space.

Reply:
> t would be shortsighted of the foundation to ignore the dominant server/build platform in the space.

I haven't said we should ignore Node.js. What I'm saying is this language targets only Node.js developers, excluding a lot of potentials members. I believe this is exactly the opposite of what we want to do here.

Reply:
Right - I’m asking who are the non-node developers being excluded? Which projects are thus impacted?

I’m not aware of any, but hopefully you can enlighten me :-)

Reply:
First paragraph of the proposal (emphasis mine):

> The Individual Membership program will bring together the ecosystem around $FOUNDATION projects, **users who are not current contributors**, that we want to feel more engaged and welcoming than **other languages**, through participation, input, and accessing the benefits of the foundation that have historically been perceived as available only to contributors.

It talks about **users** and **other languages**, not **developers** and **Node.js**.

As you know, until relatively recently only browser vendors participated at the W3C even though the community had a lot to say. They were **the users**, but didn't have a say even if they wanted (or there wasn't a clear direct way). This is the same scenario. We are excluding all the users of the projects we are creating.

Need more concrete scenarios?
A WordPress developer uses a few of the foundation projects (e.g.: jQuery, lodash, or the webhint browser extension to improve their website) and wants to get more involved with the foundation. He could very well not be a Node.js developer or not identify as one.

Another example? React Native developers. They use a lot of libraries published to the registry (which is not Node.js). Do all of them consider themselves Node.js developers? I can guarantee you that most of them don't.

I don't know what else I can say to make you see that this wording does not reach every JavaScript user or developer that could become a member, so I will stop indulging you.

Reply:
That's a fair point; although all of those React Native users are *using* node to npm install things, and the _projects_ that Wordpress dev uses were all built with node, the Wordpress dev may never use it.